src/modules/renderer.ts

```diff
@@ -157,6 +171,12 @@ class Renderer {
     // Draw it
     this.gl.drawArrays(this.gl.TRIANGLES, 0, this.modelLength / 3);
 
+    if (this.needImage) {
+      this.gl.canvas.toBlob(b => {
```

Comment:
Currently just a PNG image, but it would be smart to play around with the options a bit (like jpeg) to see what compression level would be fine.